README.md

```diff
@@ -1,302 +1,293 @@
-# Origami Tracking component
+# o-tracking
 
-Include in your product to send tracking requests to the Spoor API.
+Include in your product to send tracking requests to the [Spoor API](https://spoor-docs.herokuapp.com/).
 
-- [The spoor ecosystem](#the-spoor-ecosystem)
 - [Usage](#usage)
-- [Events](#events)
-- [Parameters](#parameters)
+- [How To View The Data](#how-to-view-the-data)
+- [Example Implementations](#example-implementations)
 - [Migration Guide](#migration-guide)
 - [Contact](#contact)
 - [Licence](#licence)
 
-## The Spoor ecosystem
-![ScreenShot](https://rawgit.com/Financial-Times/o-tracking/master/resources/images/ngda-system-design.svg)
+## Usage
 
-[Spoor API docs](https://spoor-docs.herokuapp.com/)
+Check out [how to include Origami components in your project](https://origami.ft.com/docs/components/#including-origami-components-in-your-project) to get started with `o-tracking`.
 
-## Usage
+### How to view the data
```

Comment:
I think this section might make more sense after instructions on how to setup o-tracking

Reply:
Fixed in #309